changelog/db.changelog-1.10.xml

```diff
@@ -1,33 +1,53 @@
 <?xml version="1.1" encoding="UTF-8" standalone="no"?>
 <databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
-<changeSet id ="1" author="Leigh Godson">
-    <tagDatabase tag="v1.10" />
-</changeSet>
-<changeSet id="2" author="Leigh Godson">
-    <modifyDataType tableName="delinkedCalculation" columnName="paymentBand4" newDataType="VARCHAR(4000)"/>
-</changeSet>
-<changeSet id="3" author="Leigh Godson">
-    <modifyDataType tableName="delinkedCalculation" columnName="progressiveReductions3" newDataType="VARCHAR(4000)"/>
-</changeSet>
-<changeSet id="4" author="Leigh Godson">
-    <modifyDataType tableName="delinkedCalculation" columnName="progressiveReductions4" newDataType="VARCHAR(4000)"/>
-</changeSet>
-<changeSet id="5" author="Leigh Godson">
-    <modifyDataType tableName="delinkedCalculation" columnName="referenceAmount" newDataType="VARCHAR(4000)"/>
-</changeSet>
-<changeSet id="6" author="Leigh Godson">
-    <modifyDataType tableName="delinkedCalculation" columnName="totalProgressiveReduction" newDataType="VARCHAR(4000)"/>
-</changeSet>
-<changeSet id="7" author="Leigh Godson">
-    <modifyDataType tableName="delinkedCalculation" columnName="totalDelinkedPayment" newDataType="VARCHAR(4000)"/>
-</changeSet>
-<changeSet id="8" author="Leigh Godson">
-    <modifyDataType tableName="delinkedCalculation" columnName="paymentAmountCalculated" newDataType="VARCHAR(4000)"/>
-</changeSet>
-<changeSet id="9" author="Leigh Godson">
-    <modifyDataType tableName="delinkedCalculation" columnName="applicationId" newDataType="INTEGER"/>
-</changeSet>
-<changeSet id="10" author="Leigh Godson">
-    <modifyDataType tableName="delinkedCalculation" columnName="calculationId" newDataType="INTEGER"/>
-</changeSet>
-</databaseChangeLog>
```

Comment:
Has this already been deployed up / merged to main? If so we need to do this in a new changelog, so change data types, add columns if needed, etc. Or liquibase will shout at us.

Reply:
main is still using v1.9 changelog so this should be new to it.